.travis.yml

```diff
@@ -72,11 +72,8 @@ script:
 # Deploy debian packages to bintray
 deploy:
     provider: script
-    script: beaver bintray upload -N pkg/deb/*.deb
+    script: beaver bintray upload -N deb/*.deb
```

Comment:
Should the `upload` script be called with the full path? 
Travis errors with this for the deploy step:  

```
Preparing deploy

Deploying application
sh: 1: beaver: not found

Script failed with status 127
failed to deploy
```